code/datums/custom_emotes.dm

```diff
@@ -13,7 +13,7 @@
 	if(!message)
 		return
 	if(TIMER_COOLDOWN_RUNNING(user, "custom_emotes[id]"))
-		user.balloon_alert(user, "You used that emote too recently")
+		to_chat(user, span_notice("You used that emote too recently."))
```

Comment:
Similar for this. An emote is something that happens on screen, so feedback is most useful on screen as well - a chat log of it doesn't really help anyone.